Code/Engine/RendererCore/Lights/Implementation/LightComponent.cpp

```diff
@@ -60,18 +75,56 @@ ezColorGammaUB ezLightComponent::GetLightColor() const
   return m_LightColor;
 }
 
+ezColorGammaUB ezLightComponent::GetFinalLightColor() const
+{
+  if (m_bUseColorTemperature)
+  {
+    ezColor kelvinColor;
+    kelvinColor.SetKelvin(m_uTemperature);
+    return kelvinColor;
+  }
+  else
+  {
+    return m_LightColor;
+  }
+}
+
 void ezLightComponent::SetIntensity(float fIntensity)
 {
   m_fIntensity = ezMath::Max(fIntensity, 0.0f);
 
   TriggerLocalBoundsUpdate();
 }
 
+void ezLightComponent::SetTemperature(ezUInt32 uTemperature)
+{
+  m_uTemperature = ezMath::Clamp(uTemperature, 1500u, 40000u);
+
+  InvalidateCachedRenderData();
+}
+
+ezUInt32 ezLightComponent::GetTemperature() const
+{
+  return m_uTemperature;
+}
+
 float ezLightComponent::GetIntensity() const
 {
   return m_fIntensity;
 }
 
+void ezLightComponent::SetSpecularMultiplier(float fSpecularMultiplier)
+{
+  m_fSpecularMultiplier = ezMath::Max(fSpecularMultiplier, 0.0f);
```

Comment:
clamp 0 - 1 ?

Reply:
This was intentionally left with no max so people can use it to boost specular beyond the max